recommender/models/service.py

```diff
@@ -35,3 +35,13 @@ class Service(MarketplaceDocument):
     required_services = ListField(ReferenceField("Service"))
     dataframe = BinaryField(blank=True)
     tensor = ListField(FloatField(), blank=True)
+
+    meta = {
+        "indexes": [
+            {
+                "fields": ["$name", "$description", "$tagline"],
+                "default_language": "english",
```

Comment:
This is just a question - what is this linked with (what is the list of available languages - is this a custom field, or mongo one?)